internal/application/create-code-revew.go

```diff
@@ -12,9 +12,13 @@ type Context struct {
 }
 
 func (a *App) CreateCodeReview() {
-	comment := a.aiModel.GetComment()
-	err := a.codeRepositoryProvider.CreateComment(comment)
+	changes, err := a.codeRepositoryProvider.GetPullRequestChanges()
 	if err != nil {
```

Comment:
Code Review 2024-08-18 02:28:50 - New lines 1 to 1:

	changes, err := a.codeRepositoryProvider.GetPullRequestChanges()